components/data-set-control-menu.vue

```diff
@@ -14,34 +14,49 @@
             fallback-name="placeholder"
           />
           {{ dataset.name }}
-          <UiToggle
-            :checked="dataset.visible"
-            class="data-set-control-menu__control"
-            @change="toggleLocationDataSet(dataset.id)"
-          />
+          <div class="data-set-control-menu__control">
+            <UiToggle
+              :checked="dataset.visible"
+              class="data-set-control-menu__control"
```

Comment:
why this class here?

Reply:
correct placement inside the datasetcontrolmenu